### ISSUE TITLE
[operator] replace k8s_facts with k8s_info

### ISSUE THREAD

**Issue by jmazzitelli** [kiali/kiali#2073]
```
[DEPRECATION WARNING]: The 'k8s_facts' module has been renamed to 'k8s_info'. 
This feature will be removed in version 2.13. Deprecation warnings can be 
disabled by setting deprecation_warnings=False in ansible.cfg.
```

This was something I found here: https://github.com/operator-framework/operator-sdk/issues/2168

Looks like Ansible 2.9 deprecated that k8s_facts test. We use it in our playbooks, so we need to fix this too.

**Comment by jmazzitelli** [comment]
From that operator SDK issue, it sounds like a trivial search-n-replace:

`This is pretty easy to fix; we just need to change from using k8s_facts to k8s_info. I verified with a default test setup that this works identically and drops the deprecation warnings.`

**Comment by jmazzitelli** [comment]
fixed here: https://github.com/kiali/kiali/pull/2076

**Comment by jmazzitelli** [comment]
Ansible Operator SDK moved to Ansible 2.9 in this PR: https://github.com/operator-framework/operator-sdk/pull/2203
It went in 0.13.0: https://github.com/operator-framework/operator-sdk/commit/cf1ca34ce262c45e94504083ff51ef1606a6f212

So we do not need to do this work until the productized Ansible image is at least 0.13.0 (but today it is only at 0.12.). In fact, when last I tried this, I saw problems while running with current productized ansible images. So we should wait until they release a newer base image.
